docs/development/maintenance.md

```diff
@@ -13,104 +12,124 @@ modules of ILIAS.
-	* "[Coordinator](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance-coordinator.md#coordinator-model)".
-* **"first_maintainer"**: One entry in the form `<username> (<userid>)` pointing to a valid user on 
-	https://docu.ilias.de. Only relevant if **the maintenance_model** is set to "Classic".
-* **"second_maintainer"**: One entry in the form `<username> (<userid>)` pointing to a valid user on https://docu.ilias.de. 
-	Only relevant if **the maintenance_model** is set to "Classic".
-* **"[implicit_maintainers](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance.md#implicit-maintainers)"**: 
-    An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de. Only relevant if 
-    **the maintenance_model** is set to "Classic" **and** neither a first nor a second maintainers is set.
-* **"coordinator"**: An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de.
-	Only relevant if **the maintenance_model** is set to "Coordinator".
-* **"[pr_management](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance.md#additional-competences)"** : 
-    An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de.
-* **"[issue_management](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance.md#additional-competences)"** :
-  An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de.
-* **"tester"**: One entry in the form `<username> (<userid>)` pointing to a valid user on
-  https://docu.ilias.de.
-* **"testcase_writer"**: One entry in the form `<username> (<userid>)` pointing to a valid user on
-  https://docu.ilias.de.
+# Authorities
+The ILIAS-Community strives to create and maintain a secure, reliable, and
+adaptable learning management. We foster participation by a diverse set of
+developers, but we also have to guarantee the sustainability and the quality
+of the ILIAS source code.
+
+To make sure people with diverse backgrounds and capabilities can participate
+in our Community and contribute to the development of ILIAS and its Code Base,
+we split the code into units (often called components, even though the term
+is hard to define) and we define a set of authorities community members can have
+concerning these units of code. We understand an authority as the counterpart of
+a responsibility: The people having the authorities to do something in a unit of
+code also assume the responsibility for the corresponding functions.
+
+For the context of ILIAS, we define **four** different authorities:
+
+1. **Authority to Sign of Code Changes**
+2. **Authority to Sign of Conceptual Changes**
+3. **Authority to Sign of Testcase Changes**
+4. **Authority to (De-)Assign Authorities**
+
+Each of these authorities can be held by a different set of people. This means
```

Comment:
Nebst Personen könnten hier auch Gremien/Funktionen wie das TB oder PM zum Einsatz kommen. Also bspw. die `Authority to (De-)Assign Authorities` könnte an manchen Stellen beim PM/TB liegen

Reply:
Ja, du hast Recht. Allerdings würde ich dies hier so lassen, um den Satz nicht schwerer zu machen, als unbedingt nötig (oder anders: Ich finde hier gerade keine gute Formulierung, die nicht mehr aufgibt als klärt im Kontext des nachfolgenden Satzes). Auch das TB oder der PM sind ein mögliches "set of people". Wenn jemand eine gute Idee hat, wo wir dies, jenseits der "Additional Rules" noch deutlich machen können, oder wie wir diesen Satz hier gut umbauen, dann bin ich natürlich ganz glücklich dies zu übernehmen.